test/e2e/bin_packing/bin_packing_test.go

```diff
@@ -198,8 +198,9 @@ var _ = Describe("", func() {
 			tests.LogTestClusterInfoWrapper(testDescription.FailureMessage(), []string{ns})
 			tests.LogYunikornContainer(testDescription.FailureMessage())
 		}
+
 		By("Tear down namespace: " + ns)
-		err := kClient.DeleteNamespace(ns)
+		err := kClient.TearDownNamespace(ns)
```

Comment:
Is this another bug fix? If so, could you file another JIRA to handle it?

Reply:
In `Verify_BinPacking_Node_Order_Memory`, we create jobs without waiting for pod terminating. It's better to use `TearDownNamespace` to cleanup workload before removing the namespace. Do we want to create another JIRA or just fix it in this PR?

https://github.com/apache/yunikorn-k8shim/blob/3e59ddcdd917c292635c6be0ddb8170356ae6511/test/e2e/bin_packing/bin_packing_test.go#L171-L192

Reply:
> Do we want to create another JIRA or just fix it in this PR?

yep,that is not related to this PR/issue. It would be better to have a separate ticket for it.